kousa/lib/utils/errors.ex

```diff
@@ -1,8 +1,9 @@
 defmodule Kousa.Errors do
-  def changeset_to_first_err_message(%{errors: [{_, {message, values}}]}) do
-    Enum.reduce(values, message, fn {k, v}, acc ->
+  def changeset_to_first_err_message(%{errors: [{field, {message, values}}]}) do
```

Comment:
create a separate function for return field name + error, sometimes I just want error